func-complete-check/package-lock.json

```diff
@@ -0,0 +1,2403 @@
+{
+  "requires": true,
```

Comment:
We are using yarn, so there should be a `yarn.lock` but this file should not be needed.

Reply:
removed